package com.example.rces.mapper;

import com.example.rces.dto.SubDivisionCreateDTO;
import com.example.rces.dto.SubDivisionDTO;
import com.example.rces.models.SubDivision;
import org.mapstruct.Mapper;
import org.mapstruct.ReportingPolicy;

@Mapper(componentModel = "spring", unmappedTargetPolicy = ReportingPolicy.IGNORE)
public interface SubDivisionMapper extends BaseMapper<SubDivision, SubDivisionDTO, SubDivisionCreateDTO> {



}
